app/src/androidTest/java/org/mozilla/reference/browser/ui/ThreeDotMenuTest.kt

```diff
@@ -242,4 +242,21 @@ class ThreeDotMenuTest {
             verifyAddonsView()
         }
     }
+
+    @Test
+    // Verifies the Synced tabs menu opens from a tab's 3 dot menu and displays the correct view if the user isn't signed in
+    fun openSyncedTabsTest() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        navigationToolbar {
+        }.enterUrlAndEnterToBrowser(defaultWebPage.url) {
+            mDevice.waitForIdle()
+        }.openNavigationToolbar {
+        }.openThreeDotMenu {
+            verifySyncedTabsButtonExists()
+        }.openSyncedTabs {
+            mDevice.waitForIdle()
```

Comment:
Do we need this even though the next check waits for the existence of the element for specific time? 
If would be good if we don't need this line (same as the other on line 253) within the tests. But if there is no other option, it is fine

Reply:
Took your advice and made the changes .
**✔️ Successfully re-ran 30x on Firebase**

Will make an assessment and update the other tests in the same way as well in a separate PR.
**Thank you very much for your help and guidance! ☺️**